src/components/BlueHint/index.js

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+
+import style from './style.scss';
+
+const BlueHint = ({
```

Comment:
The component shouldn't be named after its colour. It could be just `hint` and if there are a few similar components they can have modifiers e.g. 'hint--blue'.

Reply:
👍